builder/local_relay.go

```diff
@@ -105,6 +105,10 @@ func (r *LocalRelay) SubmitBlockCapella(msg *capella.SubmitBlockRequest, _ Valid
 	return r.submitBlockCapella(msg)
 }
 
+func (r *LocalRelay) Config() RelayConfig {
```

Comment:
could we add a concise comment here? specifically i'd like us to clarify why it's okay for LocalRelay to return empty RelayConfig

Reply:
> way to not specify a config for all IRelay as only RemoteRelay is using the config

We could probably add a method that exposes type of relay that we use to determine whether it has config or not.

That being said, I'm wondering why we can't put information into the config like `enableBeaconChecks`, `ForkData`, `*bls.SecretKey`, etc into the config so that it provides some value even for local relay

Reply:
yeah exactly